src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/UserInfoController.cs

```diff
@@ -6,22 +6,18 @@
 using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Authentication;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Extensions.Localization;
 using OpenIddict.Abstractions;
 using OpenIddict.Server.AspNetCore;
 using OrchardCore.Modules;
 using static OpenIddict.Abstractions.OpenIddictConstants;
 
 namespace OrchardCore.OpenId.Controllers
 {
+    // Note: the error descriptions used in this controller are deliberately not localized as
```

Comment:
I think we should move this note before setting the description

Reply:
We can certainly do that. It's worth noting that `AccessController` already has the same message at the same place:

https://github.com/OrchardCMS/OrchardCore/blob/18938aa517fa151f5ac582d03cb80c93ca2f7b27/src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs#L26-L29

If we decide to move it before every returned error, this will result in many copies. Is it fine for you?